Makefile

```
@@ -275,7 +275,7 @@ mimir-build-image/$(UPTODATE): mimir-build-image/*
 # All the boiler plate for building golang follows:
 SUDO := $(shell docker info >/dev/null 2>&1 || echo "sudo -E")
 BUILD_IN_CONTAINER ?= true
-LATEST_BUILD_IMAGE_TAG ?= pr8347-55c99be7fb
+LATEST_BUILD_IMAGE_TAG ?= pr8600-8a8fd767c7
```

```suggestion
LATEST_BUILD_IMAGE_TAG ?= pr8347-55c99be7fb
```

let's let CI update it with a new image (hoping this comment will suffice)